### PR TITLE
Add the `vnc` user

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds the [cisagov/ansible-role-vnc-server](https://github.com/cisagov/ansible-role-vnc-server) Ansible role to this AMI.

See also cisagov/cool-assessment-terraform#256.

## 💭 Motivation and context ##

Resolves cisagov/cool-system-internal#37.  (See in particular [this comment](https://github.com/cisagov/cool-system-internal/issues/37#issuecomment-2433076345) and the ones that follow.)

## 🧪 Testing ##

All automated tests pass.  I also built and tested a new staging AMI with these changes and verified that it functioned as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] Create a prerelease and test the staging AMI with these changes.

## ✅ Pre-merge checklist ##

- [x] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by dv4harr10 — APPROVED** [review]

File: src/vnc.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (dav3r) [inline src/vnc.yml:1]: Can you please add a comment in here (or in `src/playbook.yml`, wherever you feel it's more appropriate) explaining that we are installing the VNC server in order to add the VNC ssh user and not to actually use the VNC server itself?

**Review by dav3r — APPROVED** [review]
👍  though I request the addition of an explanatory comment.

File: src/vnc.yml (hunk 1 above)
Comment (jsf9k) [reply src/vnc.yml:1]: Please see commit 00cd8b1.

Comment (dav3r) [reply src/vnc.yml:1]: Thanks!

**Pull request merged**